share/rocmcmakebuildtools/cmake/ROCMTest.cmake

```diff
@@ -31,7 +31,7 @@ macro(rocm_enable_test_package NAME)
 endmacro()
 
 if(POLICY CMP0079)
-    cmake_policy(SET CMP0079 OLD)
+    cmake_policy(SET CMP0079 NEW)
 endif()
```

Comment:
Isn't it new by default if it exists?

Reply:
`cmake_minimum_required` sets this to a value matching the behaviour from that version, so the new behaviour needs to be set to work with newer versions.